modules/ossm-cr-example.adoc

```diff
@@ -18,63 +18,257 @@ This example `ServiceMeshControlPlane` definition contains all of the supported
+                termination: reencrypt
+          runtime:
+            deployment: {}
+            pod: {}
+      kiali:
+        name: kiali # kiali CR to create/reference
+        install: # install kiali CR if not present
+          config:
+            dashboard:
+              viewOnly: false
+              enableGrafana: true
+              enableTracing: true
+              enablePrometheus: true
+          service: 
+            ingress:
+              contextPath: /kiali
+          runtime:
+            deployment: {}
+            pod: {}
+----
 
-    grafana:
-      enabled: true
+The following table lists the parameters for the `ServiceMeshControlPlane` resource.
 
-    tracing:
-      enabled: true
-      jaeger:
-        template: all-in-one
-----
+.`ServiceMeshControlPlane` resource parameters
+|===
+|Name |Description |Type
+
+|`apiVersion`
+|APIVersion defines the versioned schema of this representation of an object. Servers convert recognized schemas to the latest internal value, and may reject unrecognized values. The value for {ProductName} version 2.0 is `maistra.io/v2`.
+|string
+
+|kind
+|Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated.
+|CamelCase string
+
+|metadata
+|Type of installation. You can provide a name for your deployment to keep track of your work for example, `basic-install`, or `production`.
+|string
+
+|spec
+|The specification of the desired state of this `ServiceMeshControlPlane`. This includes the configuration options for all components that comprise the control plane.
+|For more information, see `ControlPlaneSpec`.
+
+|status
+|The current status of this `ServiceMeshControlPlane` and the components that comprise the control plane. 
+|For more information, see `ControlPlaneStatus`.
+|===
+
+The following table lists the specifications for the `ServiceMeshControlPlane` resource. These parameters configure {ProductName} for your microservices and apps.
+
+.`ServiceMeshControlPlane` resource spec
+|===
+|Name |Description |Configurable parameters
+
+|addons
+|Addons is used to configure additional features beyond core control plane components, such as visualization, or metric storage.
+|`3scale`, `grafana`, `jaeger`, `kiali`, `prometheus`, and `stackdriver`
+
+|cluster
+|Cluster is the general configuration of the cluster (cluster name, network name, multi-cluster, mesh expansion, etc.)
+|`meshExpansion`, `multiCluster`, `name`, and `network`
+
+|gateways
+|Gateways configures gateways for the mesh.
+|`enabled`, `additionalEgress`, `additionalIngress`, `egress`, `ingress`, and  `openshiftRoute`
+
+|general
+|General represents general control plane configuration that does not fit anywhere else.
+|`logging` and `validationMessages`
+
+|policy
+|Policy configures policy checking for the control plane. If `Values.policy.enabled` is set to true, policy checking is enabled.
```

Comment:
I think those are remnants from the old smcp v1 values.yaml.  Those may be useful for folks converting from v1 to v2.  In this case, there is no spec.policy.enabled, as it would be spec.policy.type=Mixer or Istiod or None or Remote